tests/suites/test_suite_ssl.function

```diff
@@ -1055,15 +1054,16 @@ exit:
 }
 
 /*
- * Read application data and increase read counter if necessary.
+ * Read application data and increase read counter and fragments counter if necessary.
  */
 int mbedtls_ssl_read_fragment( mbedtls_ssl_context *ssl, unsigned char *buf,
                                int buf_len, int *read,
-                               const int expected_fragments )
+                               int *fragments, const int expected_fragments )
 {
     int ret = mbedtls_ssl_read( ssl, buf + *read, buf_len - *read );
     if( ret > 0 )
     {
+        ( *fragments )++;
```

Comment:
I meant a space between the closing bracket and `++`, but it's a minor.

Reply:
In our code you can find many places where operator `++` is used without spaces